src/lib/css-tag.ts

```diff
@@ -37,6 +44,10 @@ export class CSSResult {
   }
 }
 
+export const unsafeCss = (value: any) => {
+  return new CSSResult(String(value), constructionToken);
+};
+
 const textFromCSSResult = (value: CSSResult) => {
   if (value instanceof CSSResult) {
     return value.cssText;
```

Comment:
Consider mentioning unsafeCss in the error message here.

Reply:
Done.